module/zfs/zfeature_common.c

```diff
@@ -242,4 +242,15 @@ zpool_feature_init(void)
 	    "Support for blocks larger than 128KB.", B_FALSE, B_FALSE, B_FALSE,
 	    large_blocks_deps);
 	}
+
+	{
+	static const spa_feature_t large_dnode_deps[] = {
+		SPA_FEATURE_EXTENSIBLE_DATASET,
+		SPA_FEATURE_NONE
+	};
+	zfeature_register(SPA_FEATURE_LARGE_DNODE,
```

Comment:
@ahrens 
Thanks for mentioning this. It will also be useful for the lz4hc feature.